docs/src/content/docs/it/guides/authoring-content.md

```diff
@@ -80,7 +80,7 @@ description: Come utilizzare i link automatici di Starlight
 
 ## Introduzione
 
-Posso collegarmi alla [conclusione](#conclusione) che si trova più in basso.
+Posso collegarmi alla [mia conclusione](#conclusione) che si trova più in basso.
```

Comment:
Does the original content contain this word?

Reply:
It actually does...

https://github.com/withastro/starlight/blob/def71050fbdb9ae514b0a7c24ed2455f055fe6bd/docs/src/content/docs/guides/authoring-content.md?plain=1#L83C67-L83C67